src/main/java/com/mitchej123/hodgepodge/Common.java

```diff
@@ -5,29 +5,13 @@
 import org.apache.logging.log4j.Marker;
 import org.apache.logging.log4j.MarkerManager;
 
-import com.mitchej123.hodgepodge.config.ASMConfig;
-
 public class Common {
 
     public static final Logger log = LogManager.getLogger("Hodgepodge");
     public static final Marker securityMarker = MarkerManager.getMarker("SuspiciousPackets");
-    public static boolean thermosTainted;
     public static XSTR RNG = new XSTR();
 
     static {
         Common.log.info("Initializing Hodgepodge");
-        Common.log.info("Looking for Thermos/Bukkit taint.");
-        try {
-            Class.forName("org.bukkit.World");
-            Common.thermosTainted = true;
-            Common.log.warn(
-                    "Thermos/Bukkit detected; This is an unsupported configuration -- Things may not function properly.");
```

Comment:
RIP, going to miss this log message :'(

Reply:
Actually, I really really do like this log message - can you change it to the resource detector one and keep it around?